src/components/pages/GroomerDashboard/groomer-tabs.js

```diff
@@ -0,0 +1,77 @@
+import { Tabs, Row } from 'antd';
+import React from 'react';
+import Overview from './overview';
+import GroomerProfilePage from '../GroomerProfile/GroomerProfilePage';
+import { ProfileFormGR } from '../ProfileFormGR';
```

Comment:
If you intend to use this component later, go ahead and comment it out so the warning will be removed.  Or if you don't know if you intend to use it yet then just delete line 5.